### ISSUE TITLE
suggestion: Rename ServiceBot to BotFrameworkBot or FrameworkBot

### ISSUE THREAD

**Issue by cmayomsft** [billba/botbldr#3]
It would make it more clear that it's for the BotFrameworkAdapter.

**Comment by billba** [comment]
Yeah, that was the obvious choice -- the problem is that many people consider the SDK to be part of the "Bot Framework".

I think maybe we ought to call this adapter BotFrameworkServiceAdapter which is wordy but descriptive.
